main.go

```diff
@@ -171,17 +160,9 @@ func initConfig(args []string, fs perfTestUtils.FileSystem, exit func(code int))
 		if fileErr != nil {
 			log.Error("No readable config file found at path: ", configFilePath, " - Using default values.")
 		} else {
-			switch configurationSettings.ConfigFileFormat {
-			case "toml":
-				err := toml.Unmarshal(fileContent, &configurationSettings)
-				if err != nil {
-					log.Error("Failed to parse config file ", configFilePath, ". Error:", err, " - Using default values.")
-				}
-			default:
-				xmlError := xml.Unmarshal(fileContent, &configurationSettings)
-				if xmlError != nil {
-					log.Error("Failed to parse config file ", configFilePath, ". Error:", xmlError, " - Using default values.")
-				}
+			xmlError := xml.Unmarshal(fileContent, &configurationSettings)
+			if xmlError != nil {
+				log.Error("Failed to parse config file ", configFilePath, ". Error:", xmlError, " - Using default values.")
 			}
```

Comment:
Changes due to TOML removal. Everything else in this file is due to golint/gofmt recommendations.